core/app/components/FormBuilder/ElementPanel/ComponentConfig/RelationBlock.tsx

```diff
@@ -0,0 +1,76 @@
+import type { InputComponent } from "db/public";
+import { FormControl, FormDescription, FormField, FormItem, FormLabel, FormMessage } from "ui/form";
+import { Input } from "ui/input";
+import { usePubTypeContext } from "ui/pubTypes";
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "ui/select";
+
+import type { ComponentConfigFormProps } from "./types";
+
+export default ({ form }: ComponentConfigFormProps<InputComponent.relationBlock>) => {
+	const pubTypes = usePubTypeContext();
+	return (
+		<>
+			<FormField
+				control={form.control}
+				name="config.relationshipConfig.label"
+				render={({ field }) => (
+					<FormItem>
+						<FormLabel>Label</FormLabel>
+						<FormControl>
+							<Input
+								placeholder="Description of the selection"
+								{...field}
+								value={field.value ?? ""}
```

Comment:
This is so we don't get the error about a component going from controlled to uncontrolled. Not all of the fields have this though. I think we may want this for every text field, either that or figure out how to dynamically render default values for `config` with empty strings.

Reply:
yeah this is a really annoying one. i'm not sure what the best way of doing it is, but for these non-Form-fields i think this approach is fine. another way of doing it is by setting a "defaultValue" in the `useForm`, but this is nicer in this context